### DISCUSSION TITLE
是否可以只输入URL，3个exe自动测速选择最快的

### DISCUSSION THREAD

**Discussion by wttdgit** [LussacZheng/video-downloader-deploy#34] [General]
甚至3个exe可以3线程同时分段下载后合并

**Comment by LussacZheng** [comment]
> 是否可以只输入URL，3个exe自动测速选择最快的

目前不能，也没有计划实现该功能。

> 甚至3个exe可以3线程同时分段下载后合并

不现实，因为这涉及到进程间通信。如果你只需要单个工具的多线程下载，你可以试一试 `lux` 的 [`-n` 参数](https://github.com/iawia002/lux#multi-thread)，或调用 [aria2](https://github.com/LussacZheng/video-downloader-deploy/wiki/Aria2) 。
